third_party_integration/huggingface_transformers/0001-Modifications-for-NNCF-usage.patch

```diff
@@ -277,11 +277,11 @@ index 2bd4e90f..2b1a60e8 100644
 -                    model_to_save.save_pretrained(output_dir)
 +
 +                    model_to_save = model.module if hasattr(model, 'module') else model  # Take care of distributed/parallel training
-+                    model_to_save.save_pretrained(output_dir, saved_module_override=model_to_save)
```

Comment:
@vshampor updated hugging face patch to resume from compression state, not from legacy model's state_dict
please take a look, 

started 14 nightly build to check 3rd party integration. it worked for me locally.